modules/oadp-checking-api-group-versions.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-checking-api-group-versions_{context}"]
+= Checking Kubernetes API group versions
+
+If you use Velero to backup and restore a source cluster that has a resource that has multiple Kubernetes API versions registered to it, Velero backs up only the version of that resource that uses the `preferredVersion` of that resource's Kubernetes API.
+
+For example, if you have a resource that has 3 `apiVersions`  - `v1alpha1`, `v1beta1`, and `v1` - registered in the cluster, Velero only backs up the version of that resource that uses the `preferredVersion` of that CRD, v1. So, when Velero attempts to restore this resource, the cluster must have `v1` registered in its set of available `api-resources`.
+
+Therefore, you must check which Kubernetes API group versions you have on your source cluster.
```

Comment:
I think it should be:
Therefore, you must check which Kubernetes API group versions you have on your **target** cluster.

Reply:
Done.